index/templates/base.html

```diff
@@ -0,0 +1,263 @@
+              Email
+            </label>
+            <input class="form-control input-sm" type="text" id="inputSmall" placeholder="Email">
+
+          <div class="poptxt_sm">
+            <br><a href="" class="btn btn-default btn1">Submit</a>
+          </div>
+        </div>
+      </form>
+    </div>
+  </div>
+
+  <!-- forget_password popup -->
+  <div id="forget-popup" class="mfp-hide">
+    <div class="jumbotron popdiv">
+      <h2><p class="poptitle"><b>Forget Password</b></p></h2>
+      <form class="form-horizontal forget">
+        <div class="form-group">
+
+          <label class="control-label" for="inputSmall">
+            Enter your account and email, and we will help you !
+          </label>
+          <br>
+          <label class="control-label" for="inputSmall">Account</label>
+          <input class="form-control input-sm" type="text" id="inputSmall" placeholder="Account">
+          <label class="control-label" for="inputSmall">Email</label>
+          <input class="form-control input-sm" type="text" id="inputSmall" placeholder="Email">
+
+          <div class="poptxt_sm">
+            <br>
+            <a href="" class="btn btn-default btn1">Submit</a>
+          </div>
+        </div>
+      </form>
+    </div>
+  </div>
+
+  <!-- sidebar -->
+  <div id="mwt_mwt_slider_scroll">
+    <div id="mwt_fb_tab">
+      <a href="#feedback-popup" class="open-popup-link sidebar_color">
+        <span>F</span>
+        <span>e</span>
+        <span>e</span>
+        <span>d</span>
+        <span>b</span>
+        <span>a</span>
+        <span>c</span>
+        <span>k</span>
+      </a>  
+    </div>
+    <div id="mwt_slider_content">feedback</div>
+
+    <div id="mwt_fb_tab2">
+      <span>{{ people }}</span>      
+    </div>
+    <div id="mwt_slider_content">feedback</div>
+  </div>
+
+  <!-- navbar -->
+  <div class="navbar navbar-default newbar">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-responsive-collapse">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="/"><p class="p3">NTHU Online Judge</p></a>
+    </div>
+    <div class="navbar-collapse collapse navbar-responsive-collapse">
+      <ul class="nav navbar-nav">
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+            <p class="p2">Contest<b class="caret"></b></p>
+          </a>
+          <ul class="dropdown-menu">
+            <li><a href="#">contest archive</a></li>
+            <li><a href="#">running contest</a></li>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+            <p class="p2">Problem<b class="caret"></b></p>
+          </a>
+          <ul class="dropdown-menu">
```

Comment:
Sorry.I know.I haven't changed yet.If I changed,I would write in the commit comment.